README.md

```diff
@@ -0,0 +1,101 @@
+
+<a name="readme-top"></a>
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** git and github training project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<p>HTML, CSS</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **HTML**
+- **CSS**
+- **Linter**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] As requeted by the previous reviewer, kindly make sure to remove the `LIVE DEMO` section which will help help you t keep your repo clean because for now, you don't have the deployed version of your portfolio, you will add this section again once you deployed your portfolio. Kindly fix that.